src/Data/List/NonEmpty/Properties.agda

```diff
@@ -69,6 +72,47 @@ toList->>= f (x ∷ xs) = begin
   List.concat (List.map toList (List.map f (x ∷ xs)))
     ∎
 
+-- turning equalities of lists that are not empty to equalities on non-empty lists ...
+∷→∷⁺ : ∀ {x y : A} {xs ys : List A} →
+      (x List.∷ xs) ≡ (y List.∷ ys) →
+      (x List⁺.∷ xs) ≡ (y List⁺.∷ ys)
+∷→∷⁺ refl = refl
+
+-- ... and vice versa
+∷⁺→∷ : ∀ {x y : A} {xs ys : List A} →
+      (x List⁺.∷ xs) ≡ (y List⁺.∷ ys) →
+      (x List.∷ xs) ≡ (y List.∷ ys)
+∷⁺→∷ refl = refl
+
+------------------------------------------------------------------------
+-- _⁺++⁺_
+length-⁺++⁺ : (xs ys : List⁺ A) →
```

Comment:
Very minor but can we have an empty line between this lemm and the section heading?